pages/common/pnpm.md

```diff
@@ -0,0 +1,40 @@
+# pnpm
+
+> Fast, disk space efficient package manager for Javascript and Node.js.
+> Manage Node.js projects and their module dependencies.
+> More information: <https://pnpm.io//>.
+
+- Interactively create a `package.json` file:
+
+`pnpm init`
+
+- Download all the packages listed as dependencies in package.json:
+
+`pnpm install`
+
+- Download a specific version of a package and add it to the list of dependencies in `package.json`:
+
+`pnpm install {{module_name}}@{{version}}`
+
+- Download a package and add it to the list of dev dependencies in `package.json`:
+
+`pnpm install {{module_name}} --save-dev`
+
+`pnpm install -D {{module_name}}`
+
+- Download a package and install it globally:
+
+`pnpm install -g {{module_name}}`
+
+- Uninstall a package and remove it from the list of dependencies in `package.json`:
+
+`pnpm uninstall {{module_name}}`
+
+- Print a tree of locally-installed dependencies:
+
+`pnpm list`
+
+- List top-level globally installed modules:
+
+`pnpm list -g --depth={{0}}`
```

Comment:
Same here, is `-g` maybe some kind of globally available flag?

Reply:
yeah it is same as npm's `-g` flag

Reply:
Then we should replace it with `--global`

Reply:
nah i don think we should replace it, because almost nowhere anyone uses --global but everyone uses -g be it tutorials or anything, so there is no need to be extra verbose

Reply:
@bl-ue what do you think?

Reply:
Generally in tldr pages we prefer to use the long-form arguments, as this aids clarity - so I'd suggest using the long-form here. However, on the `npm` page we do this:

```markdown
- Download a package and install it globally:

`npm install -g {{module_name}}`
```

....so this may be ok - but I'd suggest updating the description to read `- List top-level [g]lobally installed modules:` though, just in case.

Reply:
Can you suggest the change? I'm currently on mobile so can't edit stuff easily

Reply:
[Done](https://github.com/tldr-pages/tldr/pull/5863/files#r624548363)